packages/grafana-ui/src/components/Table/TableCellInspectModal.tsx

```diff
@@ -16,7 +16,10 @@ export function TableCellInspectModal({ value, onDismiss, mode }: TableCellInspe
   if (isString(value)) {
     try {
       value = JSON.parse(value);
```

Comment:
Hmm, I bet this will show numeric values as code mode, do we want to check to see if the string begins with [ or { still? @leeoniya